src/abstractdataframe/show.jl

```diff
@@ -68,7 +68,27 @@ if VERSION < v"1.5.0-DEV.261" || VERSION < v"1.5.0-DEV.266"
     end
 end
 
-"""Return compact string representation of type T"""
+function batch_compacttype(types::Vector{Any}, maxwidths::AbstractVector{Int},
```

Comment:
Choose between `Vector` and `AbstractVector`? :-) Same below.

Reply:
I opted to use `Vector` everywhere to signal we want to avoid specialization.